### ISSUE TITLE
Explain the use of schema.org accessibilityAPI

### ISSUE THREAD

**Issue by rdeltour** [daisy/kb#1]
`schema:accessibilityAPI` is only briefly described as 

> Indicates that the resource is compatible with the referenced accessibility API.

It should be explained that only its "ARIA" value really makes sense in EPUB, and when to use it.

I'm still not sure I fully understand the usefulness of this property myself, to be honest. Should it only be used to describe that interactive scripted content is properly updating the ARIA roles dynamically? Or is it to indicate any usage of ARIA roles? Not sure how it's exactly useful in any case.

**Comment by mattgarrish** [comment]
It's effectively the equivalent of meeting [WCAG 4.1.2](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#ensure-compat-rsv) for interactive controls. States and properties also have to be maintained to be useful; it's not about roles alone.

It indicates that custom interfaces can be controlled by supporting assistive technologies, in other words, which is important information when assessing usability.

You don't have to include structural roles to meet WCAG.

**Comment by mattgarrish** [comment]
I'll tweak it when I have a minute, if it needs saying...

**Comment by rdeltour** [comment]
> You don't have to include structural roles to meet WCAG.

right, but nor do you need ARIA to meet 4.1.2, which is why I personally find this property not very informative in itself :)

**Comment by mattgarrish** [comment]
> right, but nor do you need ARIA to meet 4.1.2

If you're using standard controls, then there's no issue for anyone, yes.

But there's no way to make _custom controls_ accessible without ARIA. That's why the value is there.

**Comment by mattgarrish** [comment]
I've updated the page to add a bit more descriptive text and also link across to the techniques document, as I don't want to maintain lengthy descriptions in two places.

**Comment by mattgarrish** [comment]
Closing this issue off. We're generally discouraging the use of this property now because of the confusion it cases, so whenever the techniques are updated I'll return and note the same here.
